### ISSUE TITLE
error of werkzeug 2.2.0 version

### ISSUE THREAD

**Issue by bekishev04** [0b01001001/spectree#242]
Error with  werkzeug >= 2.2.0 version

```
Traceback (most recent call last):
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/flask/app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/flask/app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/spectree/plugins/flask_plugin.py", line 241, in <lambda>
    view_func=lambda: jsonify(self.spectree.spec),
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/spectree/spec.py", line 83, in spec
    self._spec = self._generate_spec()
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/spectree/spec.py", line 249, in _generate_spec
    path, parameters = self.backend.parse_path(
  File "/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/spectree/plugins/flask_plugin.py", line 62, in parse_path
    from werkzeug.routing import parse_converter_args, parse_rule
ImportError: cannot import name 'parse_rule' from 'werkzeug.routing' (/Users/matvei/Desktop/Work/project/.venv/lib/python3.10/site-packages/werkzeug/routing/__init__.py)
```

I found same issue here https://github.com/python-restx/flask-restx/issues/460

**Comment by kemingy** [comment]
Thanks for your feedback.

* there is a hot fix #241 

We will find a better way to fix it later.

**Comment by bekishev04** [comment]
Thanks, We will wait

**Comment by kemingy** [comment]
* fixed in #244 

Although this is not a perfect solution, but we cannot find a better way for now. I guess werkzeug won't break the schema so this should work for a while.

Close this but feel free to reopen it if you have any other questions.
